### PR TITLE
Pa/story 36

### PR DESCRIPTION
**Pull request by palworth**
Add story 36 to master

### CODE REVIEW

File: app/controllers/merchant/dashboard_controller.rb [hunk 1]
```diff
@@ -1,6 +1,4 @@
 class Merchant::DashboardController < Merchant::BaseController
   def show
-    # merchant = Merchant.find(current_user.merchant.id)
```
Comment (palworth) [inline app/controllers/merchant/dashboard_controller.rb:3]: refactor

File: app/views/merchant/dashboard/show.html.erb [hunk 2]
```diff
@@ -10,4 +10,5 @@
      <h5>Total Cost of Merchant Items: <%= number_to_currency(item_order.order.merchant_total(current_user.merchant.id)) %></h5>
    <% end %>
  <% end %>
+ <%= link_to 'Our store items',"/merchant/#{current_user.merchant.id}/items" %>
```
Comment (palworth) [inline app/views/merchant/dashboard/show.html.erb:13]: link to items for merchant

File: config/routes.rb [hunk 3]
```diff
@@ -59,6 +59,7 @@
   #only admin users will be able to reach this resource
   get '/dashboard', to: "dashboard#show"
   get '/:merchant_id/dashboard', to: "dashboard#show"
+  get '/:merchant_id/items', to: "items#index"
```
Comment (palworth) [inline config/routes.rb:62]: route to items controller within merchant

File: spec/features/admins/access_to_merchant_dashbaord_show_spec.rb [hunk 4]
```diff
@@ -92,6 +84,12 @@
       expect(page).to have_content(new_order.created_at.strftime("%Y-%m-%d"))
       expect(page).to have_content("Quantity of Items in Order: 1")
       expect(page).to have_content("Total Cost of Merchant Items: $100")
+
+      click_link 'Our store items'
+      expect(current_path).to eq("/merchant/#{@bike_shop.id}/items")
```
Comment (palworth) [inline spec/features/admins/access_to_merchant_dashbaord_show_spec.rb:89]: test for 36

**Review by agallant121 — APPROVED** [review]

**Pull request merged**